ngraph/core/src/op/reduce_logical_or.cpp

```diff
@@ -62,16 +55,25 @@ namespace
             return false;
         }
     }
-} // namespace
+} // namespace reduce_or
 
 bool op::v1::ReduceLogicalOr::evaluate(const HostTensorVector& outputs,
                                        const HostTensorVector& inputs) const
 {
     NGRAPH_OP_SCOPE(v1_ReduceLogicalOr_evaluate);
+    NGRAPH_CHECK(validate_host_tensor_vector(inputs, 2));
+    NGRAPH_CHECK(validate_host_tensor_vector(outputs, 1));
     const auto& data = inputs[0];
     const auto& axes = inputs[1];
     const auto& out = outputs[0];
-    return evaluate_reduce_logical_or(data, axes, out, get_keep_dims());
+    if (data->get_element_type() != element::boolean ||
```

Comment:
Here as well - you can use `is_integral()` instead.

Reply:
Please check the previous comment in reduce logical and